src/libraries/System.Text.Json/tests/Serialization/TestClasses/TestClasses.StructMember.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace System.Text.Json.Serialization.Tests
+{
+    public class TestClassWithStructMember : ITestClass
```

Comment:
Please add and test another version of the class where the struct members are nullable.

Reply:
Added 2 tests with `[ActiveIssue("https://github.com/dotnet/runtime/issues/36329")]`